feature-libs/customer-ticketing/root/model/customer-ticketing.model.ts

```diff
@@ -5,7 +5,13 @@ export const enum CUSTOM_CLASS {
 
 export const enum STATUS {
   OPEN = 'OPEN',
-  CLOSE = 'CLOSE',
+  CLOSE = 'CLOSED',
```

Comment:
```
STATUS {
 ID {
 OPEN='OPEN',
 CLOSE='CLOSED',
 }
 NAME {
 CLOSE='Closed',
 INPROCESS='In Process',
 }
 }
```

Reply:
got it.

Reply:
I found that typescript only supports numeric or string based enums, it is hard to use objects inside enums.